src/asana/helpers.py

```diff
@@ -386,6 +388,8 @@ def _task_description_from_pull_request(pull_request: PullRequest) -> str:
     author = _asana_user_url_from_github_user_handle(github_author.login())
     if author is None:
         author = _asana_display_name_for_github_user(github_author)
+    status_reason = _task_completion_from_pull_request(pull_request)
+    status = "closed" if status_reason.isClosed else "not closed"
```

Comment:
nit: since we're referring to Asana tasks, should this be `"complete"` and `"incomplete"`?